flink-java/src/main/java/org/apache/flink/api/java/typeutils/PojoTypeInfo.java

```diff
@@ -313,6 +319,15 @@ public int getFieldIndex(String fieldName) {
 
 	@Override
 	public TypeSerializer<T> createSerializer(ExecutionConfig config) {
+		if(config.isForceKryoEnabled()) {
+			LOG.debug("Using KryoSerializer for serializing POJOs");
```

Comment:
I still think this is the wrong place to log this, it will get repeated a gazillion times on large plans. Also, it is logged in some places on the client, and sometimes in the runtime.

It is not terribly bad, since it is debug level, but I don't get the reason to have it in the first place. Since you have no context information (what stream/dataset this is for), it is not a good debugging help anyways...